omnigibson/object_states/inside.py

```diff
@@ -20,10 +21,11 @@ def _set_value(self, other, new_value):
 
         state = og.sim.dump_state(serialized=False)
 
-        if sample_kinematics("inside", self.obj, other) and self.get_value(other):
-            return True
-        else:
-            og.sim.load_state(state, serialized=False)
+        for _ in range(os_m.DEFAULT_HIGH_LEVEL_SAMPLING_ATTEMPTS):
```

Comment:
So this high-level for loop is still needed?

Reply:
will check again

Reply:
answer: yes